### PR TITLE
Fixes #6567, tidying up glass code and #6575, #6577

### LINKED ISSUE #6567: Infinite cable coil
I used a cable coil with 5 lengths worth in it on some glass sheets by mistake, creating a glass panel surrounded by wire. I used wire cutters on it. This created a brand new cable coil with 25 lengths worth of cable in it.

### LINKED ISSUE #6575: Cryopod sprites not working as intended
They half-close for a brief second when entering it before they pop back open, making it impossible to see if anyone's inside.

And they also close properly when the person flushes and there is items to recover. If you open the pod, the recoverable item will pop out of the pod.

### LINKED ISSUE #6577: Robotic storage unit is invisible
Title. Not sure if it always is invisible or if it only goes invisible right after something stores, because I only noticed it just after something stored.
image

### PR DESCRIPTION
**Pull request by alsoandanswer**
- fixes #6567, as well as tidying up reinforced glass and wired glass crafting code.
now uses a global proc instead of a local proc, at suggestion of skull.

- fixes #6575 as well, putting in a return proc so it doesnt re-open immediately

- tweaks the cryo sprites so it aligns properly with the cosmetic tubes stuff, and fixes the path for the robot storage

- fixes direction of therapy pod.

- ups the contrast of racks a little bit to match the tables

- fixes #6577

### CODE REVIEW

**Comment by alsoandanswer** [comment]
!review

**Comment by alsoandanswer** [comment]
!bugfix

**Comment by alsoandanswer** [comment]
!wip

**Comment by alsoandanswer** [comment]
!review

**Comment by alsoandanswer** [comment]
!review

File: code/modules/materials/materials.dm [hunk 1]
```diff
@@ -532,6 +533,25 @@ var/list/name_to_material
 /material/glass/proc/is_reinforced()
 	return (hardness > 35) //todo
 
+/material/glass/wired
+	name = "wglass"
```
Comment (skull132) [inline code/modules/materials/materials.dm]:
```suggestion
	name = "wired glass"
```

File: code/modules/materials/material_sheets.dm [hunk 2]
```diff
@@ -211,6 +211,12 @@
 	icon_has_variants = TRUE
 	drop_sound = 'sound/items/drop/glass.ogg'
 
+/obj/item/stack/material/glass/wired
+	name = "wired glass"
+	icon = 'icons/obj/stacks/tiles.dmi'
+	icon_state = "glass_wire"
+	default_type = "wglass"
```
Comment (skull132) [inline code/modules/materials/material_sheets.dm]:
```suggestion
	default_type = "wired glass"
```

File: code/modules/materials/materials.dm [hunk 3]
```diff
@@ -532,6 +533,25 @@ var/list/name_to_material
...
+	icon_colour = "#00E1FF"
+	opacity = 0.3
+	integrity = 100
+	shard_type = SHARD_SHARD
+	tableslam_noise = 'sound/effects/Glasshit.ogg'
+	hardness = 40
+	weight = 30
+	stack_origin_tech = list(TECH_MATERIAL = 2)
+	composite_material = list(DEFAULT_WALL_MATERIAL = 1875,"glass" = 3750)
+	window_options = list(null)
```
Comment (skull132) [inline code/modules/materials/materials.dm]:
```suggestion
	window_options = list()
```

File: code/game/objects/items/stacks/sheets/glass.dm [hunk 4]
```diff
@@ -137,6 +108,40 @@
...
+ */
+/obj/item/stack/material/glass/wired
+	name = "wired glass tile"
+	singular_name = "wired glass floor tile"
+	desc = "A glass tile, which is wired, somehow."
+	icon = 'icons/obj/stacks/tiles.dmi'
+	icon_state = "glass_wire"
+	created_window = null
+	default_type = "wired glass"
+	construction_options = list(null)
```
Comment (skull132) [inline code/game/objects/items/stacks/sheets/glass.dm]:
```suggestion
	construction_options = list()
```

I would suspect this also is incorrect.

**Pull request merged**